routes/repo/setting.go

```diff
@@ -42,6 +42,8 @@ func Settings(c *context.Context) {
 }
 
 func SettingsPost(c *context.Context, f form.RepoSetting) {
+	c.RequireAutosize()
```

Comment:
This line should be added right after line 48 without empty line.

Reply:
done.